product_brand/reports/sale_report.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2018 Tecnativa - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import fields, models
+
+
+class SaleReport(models.Model):
+    _inherit = "sale.report"
+
+    product_brand_id = fields.Many2one(comodel_name="product.brand", string="Brand")
+
+    # pylint:disable=dangerous-default-value
+    def _query(
+        self, with_clause="", fields={}, groupby="", from_clause=""  # noqa: B006
+    ):
```

Comment:
You shouldn't (and don't need) to use this here. The Python standard would be:
```suggestion
    def _query(
        self, with_clause="", fields=None, groupby="", from_clause=""
    ):
        fields = {} or fields
```